tests/fgcmcalTestBase.py

```diff
@@ -608,7 +602,7 @@ def _testFgcmMultiFit(self, instName, testName, queryString, visits, zpOffsets):
                                        'pipelines',
                                        f'fgcmFullPipeline{instCamel}.yaml'),
                           configFiles=configFiles,
-                          inputCollections=f'{instName}/{testName}/lut,refcats/gen2',
+                          inputCollections=[f'{instName}/{testName}/lut', 'refcats/gen2'],
```

Comment:
You did the same thing above, but put a newline instead of a space before `refcats`. I'm not sure it matters, but the newline was maybe a bit more easy to parse.